src/io/utils.jl

```diff
@@ -370,6 +370,11 @@ function _bank_transformers!(data_eng::Dict{String,<:Any})
             btrans["status"] = all(tr["status"] == ENABLED for tr in trs)
             btrans["source_id"] = "transformer.$bank"
 
+            # add regulator objects if present
+            if haskey(trs[1],"controls")
+                btrans["controls"] = trs[1]["controls"]
+            end
```

Comment:
If the transformer is a bank we need to be careful of cases where only one phase of the bank is being controlled, and therefore vreg, band, ptratio, ctprim , r, and x, are different for each phase. This might mean that the controls need to be `Vector{Vector{Real}}`, like many of the transformer properties?